File: r8ted/Models/Account.cs
namespace r8ted.Models;

public class Profile
{
  public string Id { get; set; }
  public string Name { get; set; }
  public string Picture { get; set; }
  public string Banner { get; set; }
  public bool Verified {get; set;}
  
}

public class Account : Profile 
{
    public string Email { get; set; }

}

File: r8ted.client/src/models/Account.js
export class Account {
  constructor(data) {
    this.id = data.id
    this.email = data.email
    this.name = data.name
    this.picture = data.picture
    this.coverImg = data.banner
    // TODO add additional properties if needed
    this.verified = data.verified
    this.createdAt = data.created_at
  }
}
